Dev/finalpgd5.4 (#5293)
EnterpriseDB/docs: product_docs/docs/pgd/5/durability/synchronous_commit.mdx

In this example, any node in the `left_dc` group, when this commit scope is in use, will use `SYNCHRONOUS_COMMIT` to replicate changes to the other nodes in the `left_dc` group. It will look for a majority of nodes in the `left_dc` group to confirm that they have committed the transaction.

## Configuration

There are no parameters for `SYNCHRONOUS_COMMIT` and therefore no configuration.

## Confirmation

   Confirmation&nbsp;Level    | PGD Synchronous Commit Handling
-------------------------|-------------------------------

>  `received`              | A remote PGD node confirms the transaction immediately after receiving it, prior to starting the local application.

Reviewer: ```suggestion
 `received`              | A remote PGD node confirms the transaction once it's been fully received and is in in-memory write queue.
```

>  `replicated`            | Confirms after applying changes of the transaction but before flushing them to disk.

Reviewer: ```suggestion
 `replicated`            | Same behavior as `received`.
```

>  `durable`               | Confirms the transaction after all of its changes are flushed to disk.

Reviewer: ```suggestion
 `durable`               | Confirms the transaction after all of its changes are flushed to disk. Analogous to `synchronous_commit = on` in legacy synchronous replication.
```

>  `visible` (default)     | Confirms the transaction after all of its changes are flushed to disk and it's visible to concurrent transactions.

Reviewer: ```suggestion
 `visible` (default)     | Confirms the transaction after all of its changes are flushed to disk and it's visible to concurrent transactions. Analogous to `synchronous_commit = remote_apply` in legacy synchronous replication.
```

> ## Details

Reviewer 2: correct
